### PR TITLE
fix documentation links

### PR DESCRIPTION
**Pull request by gsnider2195**
# Closes: #DNE
# What's Changed

- Fix links in documentation

# TODO

- [x] Explanation of Change(s)
- n/a Attached Screenshots, Payload Example
- n/a Unit, Integration Tests
- n/a Documentation Updates (when adding/changing features)
- n/a Example Plugin Updates (when adding/changing features)
- n/a Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Review by jathanism — APPROVED** [review]

**Review by glennmatthews — APPROVED** [review]
Assuming you've validated that the updated links are valid (why doesn't mkdocs/markdownlint do this for us??) these all look like good fixes to me. :-)

File: nautobot/docs/administration/nautobot-server.md [hunk 1]
```diff
@@ -291,22 +291,22 @@ This will run the following management commands with default settings, in order:
 !!! note
     Commands listed here that are not covered in this document here are Django built-in commands.
 
-`--no-clearsessions`
```
Comment (smk4664) [inline nautobot/docs/administration/nautobot-server.md:294]: These spaces were purposefully here. Without the double-spaces, the text below will appear on the same line.

Comment (smk4664) [reply nautobot/docs/administration/nautobot-server.md:294]: image

File: nautobot/docs/administration/nautobot-server.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ $ nautobot-server dumpdata \
 
 Use this command to generate a JSON dump of the database contents.
 
-One example of using this command would be to [`export data from PostgreSQL`](../../latest/installation/migrating-from-postgresql/#export-data-from-postgresql).
+One example of using this command would be to [`export data from PostgreSQL`](../../installation/migrating-from-postgresql/#export-data-from-postgresql).
```
Comment (glennmatthews) [inline nautobot/docs/administration/nautobot-server.md:151]:
```suggestion
One example of using this command would be to [`export data from PostgreSQL`](../installation/migrating-from-postgresql.md#export-data-from-postgresql).
```

File: nautobot/docs/administration/nautobot-server.md [hunk 3]
```diff
@@ -230,8 +230,8 @@ There are a number of other options not covered here.
 
 To import the data that was exported with `nautobot-server dumpdata ...` see the following documentation:
 
-- [`Remove the auto-populated Status records from the database`](../../latest/installation/migrating-from-postgresql/#remove-the-auto-populated-status-records-from-the-mysql-database)
-- [`Import the database dump`](../../latest/installation/migrating-from-postgresql/#import-the-database-dump-into-mysql)
+- [`Remove the auto-populated Status records from the database`](../../installation/migrating-from-postgresql/#remove-the-auto-populated-status-records-from-the-mysql-database)
+- [`Import the database dump`](../../installation/migrating-from-postgresql/#import-the-database-dump-into-mysql)
```
Comment (glennmatthews) [inline nautobot/docs/administration/nautobot-server.md:234]:
```suggestion
- [`Remove the auto-populated Status records from the database`](../installation/migrating-from-postgresql.md#remove-the-auto-populated-status-records-from-the-mysql-database)
- [`Import the database dump`](../installation/migrating-from-postgresql.md#import-the-database-dump-into-mysql)
```

File: nautobot/docs/models/circuits/circuit.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ A communications circuit represents a single _physical_ link connecting exactly
 
 Each circuit is associated with a provider and a user-defined type. For example, you might have Internet access circuits delivered to each site by one provider, and private MPLS circuits delivered by another. Each circuit must be assigned a circuit ID, each of which must be unique per provider.
 
-Each circuit must be assigned to a [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/). The following statuses are available by default:
+Each circuit must be assigned to a [`status`](../../../models/extras/status/). The following statuses are available by default:
```
Comment (glennmatthews) [inline nautobot/docs/models/circuits/circuit.md:7]:
```suggestion
Each circuit must be assigned to a [`status`](../../models/extras/status.md). The following statuses are available by default:
```

File: nautobot/docs/models/dcim/cable.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ Each cable must have two endpoints defined. These endpoints are sometimes refere
 * Power outlets
 * Power ports
 
-Each cable may be assigned a type, label, length, and color. Each cable must also assigned to an operational [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/). The following statuses are available by default:
+Each cable may be assigned a type, label, length, and color. Each cable must also assigned to an operational [`status`](../../../models/extras/status/). The following statuses are available by default:
```
Comment (glennmatthews) [inline nautobot/docs/models/dcim/cable.md:16]:
```suggestion
Each cable may be assigned a type, label, length, and color. Each cable must also assigned to an operational [`status`](../../models/extras/status.md). The following statuses are available by default:
```

File: nautobot/docs/models/dcim/device.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ A device is said to be full-depth if its installation on one rack face prevents
 
 Each device must be instantiated from a pre-created device type, and its default components (console ports, power ports, interfaces, etc.) will be created automatically. (The device type associated with a device may be changed after its creation, however its components will not be updated retroactively.)
 
-Each device must be assigned a site, device role, and operational [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/), and may optionally be assigned to a specific location and/or rack within a site. A platform, serial number, and asset tag may optionally be assigned to each device.
+Each device must be assigned a site, device role, and operational [`status`](../../../models/extras/status/), and may optionally be assigned to a specific location and/or rack within a site. A platform, serial number, and asset tag may optionally be assigned to each device.
```
Comment (glennmatthews) [inline nautobot/docs/models/dcim/device.md:11]:
```suggestion
Each device must be assigned a site, device role, and operational [`status`](../../models/extras/status.md), and may optionally be assigned to a specific location and/or rack within a site. A platform, serial number, and asset tag may optionally be assigned to each device.
```

File: nautobot/docs/models/dcim/interface.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # Interfaces
 
-Interfaces in Nautobot represent network interfaces used to exchange data with connected devices. On modern networks, these are most commonly Ethernet, but other types are supported as well. Each interface must be assigned a type, an operational [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/) and may optionally be assigned a MAC address, MTU, and IEEE 802.1Q mode (tagged or access). Each interface can also be enabled or disabled, and optionally designated as management-only (for out-of-band management).
+Interfaces in Nautobot represent network interfaces used to exchange data with connected devices. On modern networks, these are most commonly Ethernet, but other types are supported as well. Each interface must be assigned a type, an operational [`status`](../../../models/extras/status/) and may optionally be assigned a MAC address, MTU, and IEEE 802.1Q mode (tagged or access). Each interface can also be enabled or disabled, and optionally designated as management-only (for out-of-band management).
```
Comment (glennmatthews) [inline nautobot/docs/models/dcim/interface.md:3]:
```suggestion
Interfaces in Nautobot represent network interfaces used to exchange data with connected devices. On modern networks, these are most commonly Ethernet, but other types are supported as well. Each interface must be assigned a type, an operational [`status`](../../models/extras/status.md) and may optionally be assigned a MAC address, MTU, and IEEE 802.1Q mode (tagged or access). Each interface can also be enabled or disabled, and optionally designated as management-only (for out-of-band management).
```

File: nautobot/docs/models/dcim/location.md [hunk 8]
```diff
@@ -9,4 +9,4 @@ Once you have defined the hierarchy of Location Types that you wish to use, you
 !!! info
     At present, Locations fill the conceptual space between the more abstract Region and Site models and the more concrete Rack Group model. In a future Nautobot release, some or all of these other models may be collapsed into Locations. That is to say, in the future you might not deal with Regions and Sites as distinct models, but instead your Location Type hierarchy might include these higher-level categories, becoming something like Country ← City ← Site ← Building ← Floor ← Room.
 
-Much like Sites, each Location must be assigned a name and operational [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/). The same default operational statuses are defined for Locations as for Sites, but as always, you can customize these to suit your needs. Locations can also be assigned to a tenant.
+Much like Sites, each Location must be assigned a name and operational [`status`](../../../models/extras/status/). The same default operational statuses are defined for Locations as for Sites, but as always, you can customize these to suit your needs. Locations can also be assigned to a tenant.
```
Comment (glennmatthews) [inline nautobot/docs/models/dcim/location.md:12]:
```suggestion
Much like Sites, each Location must be assigned a name and operational [`status`](../../models/extras/status.md). The same default operational statuses are defined for Locations as for Sites, but as always, you can customize these to suit your needs. Locations can also be assigned to a tenant.
```

File: nautobot/docs/models/dcim/powerfeed.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 
 A power feed represents the distribution of power from a power panel to a particular device, typically a power distribution unit (PDU). The power pot (inlet) on a device can be connected via a cable to a power feed. A power feed may optionally be assigned to a rack to allow more easily tracking the distribution of power among racks.
 
-Each power feed is assigned an operational type (primary or redundant) and one a [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/). The following statuses are available by default:
+Each power feed is assigned an operational type (primary or redundant) and one a [`status`](../../../models/extras/status/). The following statuses are available by default:
```
Comment (glennmatthews) [inline nautobot/docs/models/dcim/powerfeed.md:5]:
```suggestion
Each power feed is assigned an operational type (primary or redundant) and one a [`status`](../../models/extras/status.md). The following statuses are available by default:
```

File: nautobot/docs/models/dcim/rack.md [hunk 10]
```diff
@@ -14,7 +14,7 @@ A rack must be designated as one of the following types:
 * Wall-mounted frame
 * Wall-mounted cabinet
 
-Similarly, each rack must be assigned an operational [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/). The following statuses are available by default:
+Similarly, each rack must be assigned an operational [`status`](../../../models/extras/status/). The following statuses are available by default:
```
Comment (glennmatthews) [inline nautobot/docs/models/dcim/rack.md:17]:
```suggestion
Similarly, each rack must be assigned an operational [`status`](../../models/extras/status.md). The following statuses are available by default:
```

File: nautobot/docs/models/dcim/site.md [hunk 11]
```diff
@@ -2,7 +2,7 @@
 
 How you choose to employ sites when modeling your network may vary depending on the nature of your organization, but generally a site will equate to a building or campus. For example, a chain of banks might create a site to represent each of its branches, a site for its corporate headquarters, and two additional sites for its presence in two co-location facilities.
 
-Each site must be assigned a unique name and operational [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/) and may optionally be assigned to a region and/or tenant. The following operational statuses are available by default:
+Each site must be assigned a unique name and operational [`status`](../../../models/extras/status/) and may optionally be assigned to a region and/or tenant. The following operational statuses are available by default:
```
Comment (glennmatthews) [inline nautobot/docs/models/dcim/site.md:5]:
```suggestion
Each site must be assigned a unique name and operational [`status`](../../models/extras/status.md) and may optionally be assigned to a region and/or tenant. The following operational statuses are available by default:
```

File: nautobot/docs/models/ipam/ipaddress.md [hunk 12]
```diff
@@ -4,7 +4,7 @@ An IP address comprises a single host address (either IPv4 or IPv6) and its subn
 
 Like a prefix, an IP address can optionally be assigned to a VRF (otherwise, it will appear in the "global" table). IP addresses are automatically arranged under parent prefixes within their respective VRFs according to the IP hierarchy.
 
-Each IP address can also be assigned an operational [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/) and a functional role.  The following statuses are available by default:
+Each IP address can also be assigned an operational [`status`](../../../models/extras/status/) and a functional role.  The following statuses are available by default:
```
Comment (glennmatthews) [inline nautobot/docs/models/ipam/ipaddress.md:7]:
```suggestion
Each IP address can also be assigned an operational [`status`](../../models/extras/status.md) and a functional role.  The following statuses are available by default:
```

File: nautobot/docs/models/ipam/prefix.md [hunk 13]
```diff
@@ -4,7 +4,7 @@ A prefix is an IPv4 or IPv6 network and mask expressed in CIDR notation (e.g. 19
 
 Prefixes are automatically organized by their parent aggregates. Additionally, each prefix can be assigned to a particular site (optionally also to a location within the site) and a virtual routing and forwarding instance (VRF). Each VRF represents a separate IP space or routing table. All prefixes not assigned to a VRF are considered to be in the "global" table.
 
-Each prefix must be assigned a [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/) and can optionally be assigned a role. These terms are often used interchangeably so it's important to recognize the difference between them. The **status** defines a prefix's operational state. The following statuses are provided by default:
+Each prefix must be assigned a [`status`](../../../models/extras/status/) and can optionally be assigned a role. These terms are often used interchangeably so it's important to recognize the difference between them. The **status** defines a prefix's operational state. The following statuses are provided by default:
```
Comment (glennmatthews) [inline nautobot/docs/models/ipam/prefix.md:7]:
```suggestion
Each prefix must be assigned a [`status`](../../models/extras/status.md) and can optionally be assigned a role. These terms are often used interchangeably so it's important to recognize the difference between them. The **status** defines a prefix's operational state. The following statuses are provided by default:
```

File: nautobot/docs/models/virtualization/virtualmachine.md [hunk 14]
```diff
@@ -2,7 +2,7 @@
 
 A virtual machine represents a virtual compute instance hosted within a cluster. Each VM must be assigned to exactly one cluster.
 
-Like devices, each VM can be assigned a platform and/or functional role, and an operational [`status`](https://nautobot.readthedocs.io/en/latest/models/extras/status/). The following statuses are available by default:
+Like devices, each VM can be assigned a platform and/or functional role, and an operational [`status`](../../../models/extras/status/). The following statuses are available by default:
```
Comment (glennmatthews) [inline nautobot/docs/models/virtualization/virtualmachine.md:5]:
```suggestion
Like devices, each VM can be assigned a platform and/or functional role, and an operational [`status`](../../models/extras/status.md). The following statuses are available by default:
```

File: nautobot/docs/models/virtualization/vminterface.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 # Interfaces
 
-Virtual machine interfaces behave similarly to device interfaces, and can be assigned IP addresses, VLANs, an operational [`status`](https://nautobot.readthedocs.io/en/stable/models/extras/status/) and services. However, given their virtual nature, they lack properties pertaining to physical attributes. For example, VM interfaces do not have a physical type and cannot have cables attached to them.
+Virtual machine interfaces behave similarly to device interfaces, and can be assigned IP addresses, VLANs, an operational [`status`](../../../models/extras/status/) and services. However, given their virtual nature, they lack properties pertaining to physical attributes. For example, VM interfaces do not have a physical type and cannot have cables attached to them.
```
Comment (glennmatthews) [inline nautobot/docs/models/virtualization/vminterface.md:3]:
```suggestion
Virtual machine interfaces behave similarly to device interfaces, and can be assigned IP addresses, VLANs, an operational [`status`](../../models/extras/status.md) and services. However, given their virtual nature, they lack properties pertaining to physical attributes. For example, VM interfaces do not have a physical type and cannot have cables attached to them.
```

File: nautobot/docs/user-guides/getting-started/ipam.md [hunk 16]
```diff
@@ -3,12 +3,12 @@
...
 
 * Create a Regional Internet Registry
-* Create an Aggregate  
+* Create an Aggregate
 * Create assignable IP addresses
 * Assign an IP address to an Interface on a Device
 
 These type of operations fall under an umbrella called IP Address Management (IPAM).
-The Nautobot documentation [IPAM section](https://nautobot.readthedocs.io/en/latest/core-functionality/ipam/) has more detail on IPAM and each operation.
+The Nautobot documentation [IPAM section](../../../core-functionality/ipam/) has more detail on IPAM and each operation.
```
Comment (glennmatthews) [inline nautobot/docs/user-guides/getting-started/ipam.md:11]:
```suggestion
The Nautobot documentation [IPAM section](../../core-functionality/ipam/index.md) has more detail on IPAM and each operation.
```

File: nautobot/docs/user-guides/getting-started/platforms.md [hunk 17]
```diff
@@ -2,7 +2,7 @@
 
 A Platform object can hold various information about a device, such as the OS, OS version, and NAPALM driver.
 
-Further information is available in the [Platforms](https://nautobot.readthedocs.io/en/latest/models/dcim/platform/) section of the Nautobot docs.
+Further information is available in the [Platforms](../../../models/dcim/platform/) section of the Nautobot docs.
```
Comment (glennmatthews) [inline nautobot/docs/user-guides/getting-started/platforms.md:5]:
```suggestion
Further information is available in the [Platforms](../../models/dcim/platform.md) section of the Nautobot docs.
```

File: nautobot/docs/user-guides/getting-started/tenants.md [hunk 18]
```diff
@@ -2,7 +2,7 @@
 
 A 'Tenant' signifies ownership of an object in Nautobot and as such, any object may only have a single Tenant assigned.
 
-More information on Tenants can be found in the [Tenants](https://nautobot.readthedocs.io/en/latest/models/tenancy/tenant/) section of the Nautobot docs.
+More information on Tenants can be found in the [Tenants](../../../models/tenancy/tenant/) section of the Nautobot docs.
```
Comment (glennmatthews) [inline nautobot/docs/user-guides/getting-started/tenants.md:5]:
```suggestion
More information on Tenants can be found in the [Tenants](../../models/tenancy/tenant.md) section of the Nautobot docs.
```

**Review by glennmatthews — CHANGES_REQUESTED** [review]
Looks like `mkdocs` by design only validates file relative links (`../../models/extras/status.md`) and not directory relative links (`../../../models/extras/status/`), so let's have these use the validated style.

File: nautobot/docs/administration/nautobot-server.md (hunk 1 above)
Comment (jathanism) [reply nautobot/docs/administration/nautobot-server.md:294]: I don't like the idea of putting  literal `<br/>` seeing as long as we maintain proper whitespace it should be fine.

**Comment by glennmatthews** [comment]
#2274 overlaps with this but I didn't fix the `status` links that this PR includes.

File: nautobot/docs/administration/nautobot-server.md (hunk 1 above)
Comment (smk4664) [reply nautobot/docs/administration/nautobot-server.md:294]: I too don't like the literal `<br/>` and I believe that is against markdownlint.

Comment (gsnider2195) [reply nautobot/docs/administration/nautobot-server.md:294]: Can we use 2 line breaks instead of trailing whitespace or `<br>`

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**